### ISSUE TITLE
Set dependencies versions inline instead of properties in pom.xml

### ISSUE THREAD

**Issue by bertysentry** [sentrysoftware/metricshub#118]
# Problem

Some dependencies versions are currently specified as Maven properties in the main `pom.xml`. While using properties has some benefits, it also prevents automatic upgrade suggestions from **dependabot**.

# Specifications

Whenever possible and reasonable, set versions inline in `pom.xml` for all dependencies and plugins (under `<pluginManagement>` and `<dependencyManagement>`).

**Comment by NassimBtk** [comment]
Yes, except properties that are `currently used in the code base such-as `<community-connectors.version>1.0.00</community-connectors.version>`
